### PR TITLE
With steps 5 to 9

### PR DESCRIPTION
**Pull request by Devanshk9**
GCI-<Devanshk09>

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+		c.JSON(200, gin.H{
+			"message": "Hello world!",
+		})
+	})
+
+	// name parameter
+	router.GET("/greet/:name", func(c *gin.Context) {
+		name := c.Param("name")
+		c.JSON(http.StatusOK, gin.H{
+			"message": "Hello" + name,
```
Comment (f4ww4z) [inline main.go]:
```suggestion
			"message": "Hello " + name,
```

File: main.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+//hello world
```
Comment (f4ww4z) [inline main.go]: Can you write a better method comment?

File: main.go [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+func main() {
+	router := gin.Default()
+	router.GET("/hello", func(c *gin.Context) {
+		c.JSON(200, gin.H{
+			"message": "Hello world!",
+		})
+	})
+
+	// name parameter
```
Comment (f4ww4z) [inline main.go]: Describe the endpoint in this comment, not just `name parameter`.

File: main.go [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+//hello world
+func main() {
+	router := gin.Default()
+	router.GET("/hello", func(c *gin.Context) {
```
Comment (f4ww4z) [inline main.go]: Describe the endpoint above this line, in a comment.

File: main.go (hunk 2 above)
Comment (Devanshk9) [reply main.go]: Done

File: main.go [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+	"github.com/gin-gonic/gin"
+)
+/* This is a Go server with 2 endpoints /hello & /greet/:name and with gin dependencies for 
+Google Code-in 2019 task */
+
+//This will create a JSON /hello endpoint which need to return the message as "Hello world"
+func main() {
+	router := gin.Default()
+	router.GET("/hello", func(c *gin.Context) {
+		c.JSON(200, gin.H{
```
Comment (f4ww4z) [inline main.go]:
```suggestion
	        c.JSON(http.StatusOK, gin.H{
```
For consistency.

Comment (Devanshk9) [reply main.go]: Done @f4ww4z (On the main file)

**Comment by Devanshk9** [comment]
Sorry I was not knowing that

**Comment by Devanshk9** [comment]
Hi @f4ww4z , tried to squash the commits but it was giving error When I enter git rebase -i HEAD~7 it gave an error.

**Comment by Devanshk9** [comment]
ERROR GIT

**Comment by Devanshk9** [comment]
tried several times with head 6,5 also

**Pull request merged**